src/main/java/uk/gov/hmcts/ccd/domain/service/callbacks/CallbackService.java

```diff
@@ -43,22 +47,8 @@ public CallbackService(final SecurityUtils securityUtils,
         this.defaultRetries = applicationParams.getCallbackRetries();
     }
 
-    public Optional<CallbackResponse> send(final String url,
-                                           final List<Integer> callbackRetries,
-                                           final CaseEvent caseEvent,
-                                           final CaseDetails caseDetails) {
-        return send(url, callbackRetries, caseEvent, null, caseDetails);
-    }
-
-    public Optional<CallbackResponse> send(final String url,
-                                           final List<Integer> callbackRetries,
-                                           final CaseEvent caseEvent,
-                                           final CaseDetails caseDetailsBefore,
-                                           final CaseDetails caseDetails) {
-
-        return send(url, callbackRetries, caseEvent, caseDetailsBefore, caseDetails, false);
-    }
-
+    // The retry will be on seconds T=1 and T=3 of the initial call fails at T=0
+    @Retryable(value = {CallbackException.class}, maxAttempts = 3, backoff = @Backoff(delay = 1000, multiplier = 3))
     public Optional<CallbackResponse> send(final String url,
                                            final List<Integer> callbackRetries,
```

Comment:
parameter not needed for now. I would remove it and reintroduced it later if needed

Reply:
removed this parameter along with unused constructor param `applicationParams` updated client codes & tests accordingly